imperative/python/megengine/functional/tensor.py

```diff
@@ -104,38 +104,35 @@ def full(
     dtype=None,
     device=None,
 ) -> Tensor:
-    r"""Creates a tensor of shape ``shape`` filled with ``value``.
+    r"""Return a tensor of shape ``shape`` filled with ``value``.
```

Comment:
```suggestion
    r"""Returns a new tensor having a specified ``shape`` and filled with ``value``.
```

- 你的 Return 少了个 s, 我们要求按照 Google 风格指南，使用描述性风格而不是命令式风格，需要加上。
- 连续出现了两个 shape
- 需要参考参考 《 数组 API 标准 》中所定义的规格和约定提供文档字符串。


Refs:

https://megengine.org.cn/doc/stable/zh/development/docs/python-docstring-style-guide.html